src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -1229,3 +1230,42 @@ def _configure_security_policy_storage_params(arg_ctx):
                    id_part='name')
 
         c.argument('resource_group_name', arg_type=resource_group_name_type)
+
+    ###############################################
+    #             sql instance failover-group     #
+    ###############################################
+
+    with self.argument_context('sql instance-failover-group') as c:
+        c.argument('instance_failover_group_name',
+                   options_list=['--name', '-n'],
+                   help="The name of the Instance Failover Group")
+
+        c.argument('failover_group_name',
+                   options_list=['--name', '-n'],
+                   help="The name of the Instance Failover Group")
+
+        c.argument('managed_instance',
+                   arg_type=managed_instance_param_type,
+                   options_list=['-s'])
```

Comment:
Why does this only have a short option?

Reply:
More importantly, why does this not line up with param names in other command groups?

```
Command
    az sql midb show : Get the details for a managed database.

Arguments

Resource Id Arguments
    --ids                   : One or more resource IDs (space-delimited). If provided, no other
                              'Resource Id' arguments should be specified.
    --managed-instance --mi : Name of the Azure SQL managed instance.
```